static/sass/_pattern_strip.scss

```diff
@@ -1,18 +1,201 @@
 // sass-lint:disable no-color-literals hex-notation
 @mixin anbox-p-strip {
+  @include anbox-p-strip-suru-accent;
   @include anbox-p-strip-suru-topped;
+  @include anbox-p-strip-suru-image;
+}
+
+@mixin anbox-p-strip-suru-accent {
```

Comment:
Is the anbox suru different from vanilla one?

Reply:
I'm not sure how to import surus from vanilla? the class didn't work unless I added the styles directly. Happy to update this though if I'm missing something

Reply:
Removed the `p-strip--suru-accent` scss and added the colour variables - thanks! (and removed `p-strip--suru-topped` as this isn't used anymore)